config/_default/menus/main.en.yaml

```diff
@@ -5796,16 +5796,21 @@ menu:
       parent: csm_agentless
       identifier: csm_agentless_deployment_methods
       weight: 102
+    - name: IAC Scanning
```

Comment:
@drichards-87 one nit here, let's lowercase the "a" in IaC, thanks.

Reply:
I'll update this in the PR.